src/views/profile-view/index.tsx

```diff
@@ -1,58 +1,47 @@
-import useAuthentication from 'hooks/use-authentication'
-import Button from 'components/button'
-import Heading from 'components/heading'
+import { useAllBookmarks, useBookmarkMutations } from 'hooks/bookmarks'
 import BaseNewLayout from 'layouts/base-new'
 import AuthenticatedView from 'views/authenticated-view'
-import { ProfileViewProps } from './types'
-import s from './profile-view.module.css'
+import Heading from 'components/heading'
 
-/**
- * The exported view component that handles wrapping the view content in
- * `AuthenticatedView`, which automatically handles rendering gated content.
- */
-const ProfileView = ({ user }: ProfileViewProps) => {
+const ProfileView = () => {
 	return (
 		<AuthenticatedView>
-			<ProfileViewContent user={user} />
+			<ProfileViewContent />
 		</AuthenticatedView>
 	)
 }
 
 /**
  * The content of the ProfileView that is gated behind authentication.
  */
-const ProfileViewContent = ({ user }: ProfileViewProps) => {
```

Comment:
We will still need the user data for this view. Is there a reason you removed it?

Reply:
Actually let me check the designs again, maybe im getting wires crossed with Learn